src/firstboot/firstboot.c

```diff
@@ -1496,10 +1496,15 @@ static int parse_argv(int argc, char *argv[]) {
 
         if (arg_delete_root_password && (arg_copy_root_password || arg_root_password || arg_prompt_root_password))
                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
-                                       "--delete-root-password cannot be combined with other root password options");
+                                       "--delete-root-password cannot be combined with other root password options.");
 
         if (arg_image && arg_root)
-                return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Please specify either --root= or --image=, the combination of both is not supported.");
+                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
+                                       "--root= and --image= cannot be used together.");
+
+        if (!sd_id128_is_null(arg_machine_id) && !(arg_image || arg_root) && !arg_force)
```

Comment:
What's with the --force check here? We should refuse this combo unconditionally of that no?

Reply:
In the commit message:
> If --force is specified, it works. So let's allow that in case people want to do crazy things.

I don't have a strong opinion. I can drop this if you think it's too much.

Reply:
I would drop this, let's leave --force for overriding files if they already exist